### ISSUE TITLE
Alacritty not starting with -c arg on shell

### ISSUE THREAD

**Issue by lucasqueiroz** [alacritty/alacritty#8005]
I have configured Alacritty to run a command when initializing, through the `shell` options, at `alacritty.toml`.
The configuration is the following:

```toml
import = ["~/.config/alacritty/catppuccin-frappe.toml"]

[window]
decorations = "None"

[font]
normal.family = "FiraMono Nerd Font Mono"

[shell]
program = "/bin/zsh"
args = ["-l", "-c", '"echo $TERM"']
```

If I remove the `-c` and `'"echo $TERM"'` args, it initializes correctly. With absolutely any command after `-c`, it does not.
I have tried with single + double quotes, single + single quotes with escaping, double + double quotes with escaping.

Running the same command in the terminal works as expected:

```
❯ /bin/zsh -l -c "echo $TERM"
xterm-256color
```

### System

OS: Ubuntu 24.04
Version: 0.13.2

### Logs

```
Created log file at "/tmp/Alacritty-14739.log"
[0.000005030s] [INFO ] [alacritty] Welcome to Alacritty
[0.000134271s] [INFO ] [alacritty] Version 0.13.2 (bb8ea18e)
[0.000148959s] [INFO ] [alacritty] Running on X11
[0.000478305s] [INFO ] [alacritty] Configuration files loaded from:
                                     "/home/lucas/.config/alacritty/alacritty.toml"
                                     "/home/lucas/.config/alacritty/catppuccin-frappe.toml"
[0.001781893s] [INFO ] [alacritty] winit event: NewEvents(Init)
[0.001820124s] [INFO ] [alacritty] winit event: Resumed
[0.022296790s] [INFO ] [alacritty] Using GLX 1.4
[0.062164872s] [INFO ] [alacritty] Window scale factor: 2
[0.158057291s] [INFO ] [alacritty] Running on NVIDIA GeForce RTX 2060/PCIe/SSE2
[0.158098258s] [INFO ] [alacritty] OpenGL version 3.3.0 NVIDIA 535.171.04, shader_version 3.30 NVIDIA via Cg compiler
[0.158108827s] [INFO ] [alacritty] Using OpenGL 3.3 renderer
[0.163304194s] [INFO ] [alacritty] Cell size: 18 x 37
[0.163351252s] [INFO ] [alacritty] Padding: 0 x 0
[0.163360930s] [INFO ] [alacritty] Width: 800, Height: 600
[0.229042899s] [INFO ] [alacritty] PTY dimensions: 16 x 44
[0.233256550s] [INFO ] [alacritty] Initialisation complete
[0.233329116s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(71303173)), event: Ime(Enabled) }
[0.233562161s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(71303173)), event: Resized(PhysicalSize { width: 800, height: 600 }) }
[0.233601184s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(71303173)), event: Moved(PhysicalPosition { x: 0, y: 64 }) }
[0.233656117s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(71303173)), event: RedrawRequested }
[0.242245752s] [INFO ] [alacritty] winit event: AboutToWait
[0.242317366s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 959, tv_nsec: 921706445 }, requested_resume: Some(Instant { tv_sec: 959, tv_nsec: 938315397 }) })
[0.242338164s] [INFO ] [alacritty] winit event: UserEvent(Event { window_id: Some(WindowId(WindowId(71303173))), payload: Terminal(Wakeup) })
[0.242353683s] [INFO ] [alacritty] winit event: UserEvent(Event { window_id: Some(WindowId(WindowId(71303173))), payload: Terminal(ChildExit(127)) })
[0.242365506s] [INFO ] [alacritty] winit event: UserEvent(Event { window_id: Some(WindowId(WindowId(71303173))), payload: Terminal(Exit) })
[0.244451477s] [INFO ] [alacritty] winit event: UserEvent(Event { window_id: Some(WindowId(WindowId(71303173))), payload: Terminal(Wakeup) })
[0.244506179s] [INFO ] [alacritty] winit event: AboutToWait
[0.244518482s] [INFO ] [alacritty] winit event: LoopExiting
[0.246634089s] [INFO ] [alacritty] Goodbye
Deleted log file at "/tmp/Alacritty-14739.log"
```

**Comment by chrisduerr** [comment]
Obviously it doesn't work, your shell doesn't accept any stdin but just uses the `-c` argument as stdin and quits after executing that.

**Comment by lucasqueiroz** [comment]
> Obviously it doesn't work, your shell doesn't accept any stdin but just uses the `-c` argument as stdin and quits after executing that.

There is no reason to be so rude. It is not only the `echo` cmd that does not work, it's any command at all. The initial problem was executing tmux, as used in multiple examples in this same repository, and it did not work, so I changed commands to try and understand what was going on.

I believe it was caused by powerlevel10k's script, so this can be closed still.
